packages/pluggableWidgets/accordion-native/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "accordion-native",
+    "widgetName": "Accordion",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/mendix/widgets-resources.git"
+    },
+    "config": {},
+    "packagePath": "com.mendix.widget.native",
+    "scripts": {
+        "start": "pluggable-widgets-tools start:native",
+        "build": "pluggable-widgets-tools build:native",
+        "release": "pluggable-widgets-tools release:native",
+        "lint": "..\"/../../node_modules/.bin/eslint\" --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
```

Comment:
What's going on here?

Reply:
According to Diego the double quotes are used to make the commands work on Windows. It's the same in every widget :)

Reply:
ok, I'll take the discussion to Slack, since it's not specific to this MR

Reply:
https://github.com/mendix/widgets-resources/pull/703 This finishes the discussion